src/Education/XamlGenerator/RuleHelpXamlTranslator.cs

```diff
@@ -0,0 +1,454 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+
+using static Microsoft.VisualStudio.Threading.AsyncReaderWriterLock;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Xml;
+using System.Collections.Generic;
+using System;
+
+namespace SonarLint.VisualStudio.Education.XamlGenerator
+{
+    public interface IRuleHelpXamlTranslator
+    {
+        string TranslateHtmlToXaml(string htmlContent, bool supportsInline = false);
```

Comment:
`supportsInline` is the name that is used internally inside the class. However, it's perhaps not the best name to use on the public API. Can you think of something more useful/descriptive to help callers?